layouts/neighbor.jsx

```diff
@@ -0,0 +1,121 @@
+import React, { useContext } from 'react';
```

Comment:
This is the only file that is `.jsx`. Is there some reasoning behind that?

Reply:
I would suggest adding eslint to the project as well.

Reply:
Good call on eslint, dunno how I didn't get that when I did next create app. Will add that!

And nope, just that I changed all of the other files to .js late last night in a fear that someone new to react (but not javascript) might ignore those files because of the extension